### DISCUSSION TITLE
Multiple search terms

### DISCUSSION THREAD

**Discussion by johanboekhoven** [aminalaee/sqladmin#543] [Q&A]
I looked in the documentation and in the code, but I don't think we can find use multiple terms in the search field separated by space, comma or some AND term right? 

Example I want to find items in the database with the names starting 123 and ABC, so I type 123 ABC, this currently doesn't work, any other variation also doesn't work.

Can the code somewhat easily be extended to do it?

Thanks!

**Comment by aminalaee** [comment]
Hi @johanboekhoven 
Yes I think the change should be mostly UI but this can be easily added in SQLAdmin.
Feel free to create a PR if you made on progress on this.

**Reply by johanboekhoven** [reply]
Thanks @aminalaee!
I'll try, but it will take some time before I'll get to it. If you have any quick pointers what files to look into, I see models.py >> ModelView.search_query() would that be a good starting place?

**Reply by johanboekhoven** [reply]
@aminalaee that was a lot easier than expected, I've put up a PR for you to review 🍺
